Fix issues 42 and 43

PhilPJL opened the pull request:
Erik.  Fixed my issue by creating the context file in a unique sub-folder.
Also fixed issue 42.
Ran Resharper over the solution and fixed a load of analyzer warnings.

ErikEJ commented:
Thanks, I really appreciate your contribution. Please split into three PRs

ErikEJ commented:
Don't worry about the build failure, I think it is related to the build server using VS 15.8, I Will fix that and let you know, so you can rebase

PhilPJL commented:
Erik,

 

I’ve fixed the unit test project and updated my fork.  I can create another pull request.

 

I think it’s too late to split what I’ve done though.  I got carried away 😊

 

Phil

 

From: Erik Ejlskov Jensen <notifications@github.com> 
Sent: 17 October 2018 20:04
To: ErikEJ/EntityFramework6PowerTools <EntityFramework6PowerTools@noreply.github.com>
Cc: PhilPJL <github@pjlconsultants.co.uk>; Author <author@noreply.github.com>
Subject: Re: [ErikEJ/EntityFramework6PowerTools] Fix issues 42 and 43 (#44)

 

Thanks, I really appreciate your contribution. Please split into three PRs

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/ErikEJ/EntityFramework6PowerTools/pull/44#issuecomment-430750393> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AAQlKI-fBUT5MVLd-1IPsk6QqTcb9Sf6ks5ul38pgaJpZM4Xk_km> .  <https://github.com/notifications/beacon/AAQlKHS9T7FpjMyrKCAiwipvxvynd4znks5ul38pgaJpZM4Xk_km.gif>
